pkgs/desktops/cinnamon/cinnamon-common/default.nix

```diff
@@ -74,7 +74,7 @@ stdenv.mkDerivation rec {
       setproctitle
       pygobject3
       pycairo
-      python3.pkgs.xapp # The scope prefix is required
+      xapp # The scope prefix is required
```

Comment:
```suggestion
      xapp
```

Reply:
I will drop 5efc2f82372840cbb3959c7bd856304e6e776bd0 very soon as it breaks the cinnamon test (which is still running and I expect ofborg will fail with timeout) :-) https://github.com/NixOS/nixpkgs/pull/189824#issuecomment-1236741751

For ofborg-eval-lib-tests failure that is #189826

Reply:
ah ok